addons/mail/static/src/models/thread_needaction_preview_view.js

```diff
@@ -34,6 +35,14 @@ registerModel({
                 ['res_id', '=', this.thread.id],
             ]);
         },
+        /**
+         * @private
+         * @returns {boolean}
+         */
+         _computeIsEmpty() {
+            return (!this.thread.lastNeedactionMessageAsOriginThread) ||
+                (mailUtils.htmlToTextContentInline(this.thread.lastNeedactionMessageAsOriginThread.prettyBody).length === 0);
```

Comment:
I believe that multi-lines conditions should look something like this according to our guidelines:
```suggestion
            return (
                !this.thread.lastNeedactionMessageAsOriginThread) ||
                mailUtils.htmlToTextContentInline(this.thread.lastNeedactionMessageAsOriginThread.prettyBody).length === 0
            );
```